packages/redux-subspace-observable/package.json

```diff
@@ -34,19 +34,18 @@
     "@types/es6-shim": "^0.31.34",
     "@types/node": "^8.0.47",
     "redux-subspace": "^2.4.0",
-    "rxjs": "^5.5.2"
+    "rxjs": "^5.5.10"
   },
   "peerDependencies": {
     "redux": "^3.0.0",
-    "redux-observable": "^0.16.0"
+    "redux-observable": "^0.18.0"
```

Comment:
Traditionally this would technically make it a breaking change.

Reply:
Is that because alpha versions are can be breaking with minor bumps?  If it was, for example `^1.16.0` -> `1.18.0`, would that also traditionally be considered breaking? (I have never thought of minor bumps in dependencies as breaking changes).

Reply:
My understanding is that if someone is following semver, pre 1.0 bumps of any kind should be considered unstable, but that maintainers will try to use minor bumps to try and gate potentially breaking changes. So 0.16.0 -> 0.18.0 could be breaking. But 1.16.0 -> 1.18.0 should not be. In this specific case of redux-observable there [weren't any known breaking changes between 0.16.0 and 0.18.0](https://github.com/redux-observable/redux-observable/blob/master/CHANGELOG.md#0180-2018-02-07) other than type definition changes, which IMO are almost always "breaking", however I would still consider this peer dep bump breaking as if a consumer has locked their redux-observable version to 0.16.0, they'll now get a peer deps error on install, even though they could probably fairly easily upgrade.

Unless there is a big reason against, I would personally advocate for subspace skipping 0.18.0 and just working towards v1.0.0 and rxjs v6.

Reply:
I agree with that. This PR is mostly just prep for that